brench/brench.py

```diff
@@ -117,7 +128,9 @@ def brench(config_path, files, jobs):
                 # Check correctness.
                 if first_out is None:
                     first_out = stdout
-                elif stdout != first_out and not status:
+                elif not compare_output(stdout, first_out) and not status:
+                    print(first_out)
+                    print(stdout)
```

Comment:
Looks like some stray debug printouts---probably delete these?

Reply:
oops pushed the wrong version!